secator/tasks/httpx.py

```diff
@@ -68,6 +73,9 @@ class httpx(Http):
 	proxy_socks5 = True
 	proxy_http = True
 	profile = 'cpu'
+	# Overriding the output_type to add Subdomains
+	output_types = [Url, Subdomain]
+	founded_domain_in_certificates = []
```

Comment:
this will cause issues (remote Celery run, task chunking, multiple tasks instantiations, etc...). Class attributes are not meant to be the task memory. Use the `on_init` hook and set `self.cert_domains = []` instead.